scripts/viz/param_sweep_viz.R

```diff
@@ -0,0 +1,74 @@
+# Pan-Cancer Variational Autoencoder
+# Gregory Way 2017
+#
+# Visualizing results of parameter sweep
+
+library(readr)
+library(ggplot2)
```

Comment:
Is there a reason you're opting to load these packages here rather than going with the double colon nomenclature? Can be a little clunky for plotting certainly.